server/routes/bookedtoday/arrivals/state.ts

```diff
@@ -8,16 +8,29 @@ export type NewArrival = {
   prisonNumber?: string
   pncNumber?: string
   sex?: string
+  // TODO: update this name to 'imprisonmentStatusCode'
   code?: string
   imprisonmentStatus?: string
   movementReasonCode?: string
+  expected: boolean
 }
 
 export const NewArrivalCodec: Codec<NewArrival> = {
-  write: (value: NewArrival): Record<string, string> => ({ ...value }),
+  write: (value: NewArrival): Record<string, string> => ({
+    firstName: value.firstName,
+    lastName: value.lastName,
+    dateOfBirth: value.dateOfBirth,
+    expected: value.expected.toString(),
+    ...(value.prisonNumber !== undefined && { prisonNumber: value.prisonNumber }),
+    ...(value.pncNumber !== undefined && { pncNumber: value.pncNumber }),
+    ...(value.sex !== undefined && { sex: value.sex }),
+    ...(value.code !== undefined && { code: value.code }),
+    ...(value.imprisonmentStatus !== undefined && { imprisonmentStatus: value.imprisonmentStatus }),
+    ...(value.movementReasonCode !== undefined && { movementReasonCode: value.movementReasonCode }),
+  }),
```

Comment:
now being more careful around writing cookie to state - previously we would push anything through. 
This meant we were incorrectly setting cro number in the cookie. Fixed this.